presto-docker/bin/run-presto

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+if [[ ! -d /usr/lib/presto/etc ]]; then
```

Comment:
We should probably use `/etc/presto` as that's the expected location on UNIX. Though we should also see what paths the RPM uses, as it might be better to be consistent

Reply:
RPM uses /etc/presto to store config files (https://github.com/prestosql/presto/blob/4f0197c2ca30ab994390b22e48a14caae342205f/presto-server-rpm/pom.xml#L165-L166)
and /etc/presto/catalog to store catalogs (https://github.com/prestosql/presto/blob/106e35b35875f6b63155903b65b310a57c2df629/presto-server-rpm/src/main/resources/dist/config/node.properties#L4)

Reply:
So using the RPM directly turns out to be sub-optimal, primarily as the RPM is geared to a running machine with a full OS and process stack. Instead I've added `/etc/presto` as a possible mount point for the `run-presto` command. This allows for mounting to either `/etc/presto` or `/usr/lib/presto/etc` with the latter taking preference.

Reply:
> allows for mounting to either /etc/presto or /usr/lib/presto/etc with the latter taking preference.

what do we need this flexibility for?
i am concerned this will only be confusing (even if you precisely you document which one wins, as you did)

Reply:
So the RPM works by creating a symlink between `/etc/presto` and /usr/lib/presto/etc`, further it's desirable to mimic the RPM behavior and support `/etc/presto` as a possible config location. I can't stop someone from mounting to `/usr/lib/presto/etc` so supporting both makes the most sense.

Reply:
I don't know why RPM creates this symlink (`/usr/lib/presto/etc`). It doesn't seem to be needed. 
(`launcher` will create a symlink in "data dir", which is `/var/lib/presto/data` when installed via RPM, but that's a different story)

Reply:
The launcher uses `/usr/lib/presto/etc` as the `etc` directory (technically `[install_path]/etc`) unless `--etc-dir` is specified. See `launcher.py` lines 415, 413, 406, and 24-31.